src/styles/about.scss

```diff
@@ -1,11 +1,11 @@
 @import 'learn.scss';
 
 .side-nav-about {
-  @extend .side-nav;
+  @extend %side-nav;
   width: auto;
 
   &.side-nav--open {
-    @extend .side-nav--open;
+    @extend %side-nav--open;
```

Comment:
This one seems odd, since it's extending the same selector

Reply:
Yeah, no broken styles founds after removing this extend